src/main/java/dev/dokan/dokan_java/conv/MaskValueEnum.java

```diff
@@ -0,0 +1,21 @@
+package dev.dokan.dokan_java.conv;
+
+
+public interface MaskValueEnum extends EnumInteger {
+
+    static <T extends Enum<T> & MaskValueEnum> T enumFromInt(final int value, final T[] enumValues) {
+        return EnumInteger.enumFromInt(value, enumValues);
+    }
+
+    static <T extends Enum<T> & MaskValueEnum> MaskValueSet<T> getSetFromInt(final int intValue, final T[] allEnumValues) {
+        return MaskValueSet.getSetFromInt(intValue, allEnumValues);
+    }
+
+    static <T extends Enum<T> & MaskValueEnum> MaskValueSet<T> getSetFromInt(final int intValue, final Class<T> type) {
+        return MaskValueSet.getSetFromInt(intValue, type);
+    }
+
+    default int getMaskValue() {
```

Comment:
Also here, the enum is singular, therefore this mask should maybe renamed to intValue

Reply:
Resolved with commit 3114b10f633ea2c173307afff6119c208ba7d791